src/map/conquest_system.cpp

```diff
@@ -51,10 +51,13 @@ namespace conquest
         zoneutils::ForEachZone([](CZone* PZone)
         {
             // only find chars for zones that have had conquest updated
-            if (PZone->GetRegionID() <= REGION_TYPE::TAVNAZIA)
+            if (PZone->GetRegionID() <= REGION_TYPE::DYNAMIS)
```

Comment:
Should this be < and not <=?

Reply:
Dynamis should be included (I think, because all of those zone scripts have onConquestUpdate bindings)